python/tskit/trees.py

```diff
@@ -5510,7 +5552,10 @@ def genetic_relatedness(
         :param bool span_normalise: Whether to divide the result by the span of the
             window (defaults to True).
         :param bool proportion: Whether to divide the result by the number of
-            segregating sites (defaults to True).
+            segregating sites (defaults to True). Note that this counts sites
+            that are segregating in *any* of the samples of *any* of the sample
+            sets (rather than segregating in all of the samples of the tree
+            sequence).
```

Comment:
This should say "segregating between" instead of "segregating in". Also, maybe it should say
```
Whether to divide the result by :meth:`.segregating_sites`, called with the same ``windows`` and ``mode`` (defaults to True).
```
... since if mode='branch' then it's not the *number* of segregating sites.